src/AppInstallerCLICore/Command.cpp

```diff
@@ -313,7 +280,7 @@ namespace AppInstaller::CLI
                 {
                     auto feature = ExperimentalFeature::GetFeature(command->Feature());
                     AICLI_LOG(CLI, Error, << "Trying to use command: " << *itr << " without enabling feature " << feature.JsonName());
-                    throw CommandException(Resource::String::FeatureDisabledMessage, feature.JsonName());
+                    throw CommandException(Resource::String::FeatureDisabledMessage(Utility::LocIndView{ feature.JsonName() }));

```

Comment:
Should we change the return type of `JsonName()` to LocIndString/View instead of wrapping occurrences?
Question applies to all similar cases in this PR.